packages/matter.js/tools/mom/chip/load-chip.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Directory, Repo } from "../../util/github.js";
+import { readFileWithCache } from "../../util/file.js";
+import { readFileSync } from "fs";
+import { homedir } from "os";
+import { resolve } from "path";
+import { Logger } from "../../../src/log/Logger.js";
+import { JSDOM } from "jsdom";
+import { AnyElement, ClusterElement, DatatypeElement, Globals, Metatype, ValueElement } from "../../../src/model/index.js";
+import { translateChip } from "./translate-chip.js";
+
+const AUTH_FILE = resolve(homedir(), ".gh-auth");
+
+const logger = Logger.get("load-chip");
+
+const auth = await loadAuth();
+const repo = new Repo("project-chip", "connectedhomeip", "v1.1-branch", readFileWithCache, auth);
```

Comment:
ooohhh you already use v1.1 branch? but theeeennn there should be not that product appearance attribute

Reply:
Yeah see previous comment, it's in there